EventFilter/L1TRawToDigi/src/implementations_stage2/GlobalAlgBlkUnpacker.cc

```diff
@@ -77,8 +77,8 @@ namespace stage2 {
          if( (block.header().getID()!=initialBlkID+4 && block.header().getID()!=prescaledBlkID+4 && block.header().getID()!=finalBlkID+4 ) || wd<4) {
            for(unsigned int bt=0; bt<32; bt++) {
 	     int val = ((raw_data >> bt) & 0x1);
-	     int algBit = bt+wd*32+algOffset;
-             if(val==1 && algBit < 128) { //FIX ME...get dimension from object
+	     unsigned int algBit = bt+wd*32+algOffset;
+             if(val==1 && algBit < alg.maxPhysicsTriggers) { //FIX ME...get dimension from object
```

Comment:
will this affect the behaviour of the HLTL1TSeed modules when we run over data with CMSSW_8_0_1 ?